calico/getting-started/kubernetes/managed-public-cloud/aks.mdx

```diff
@@ -26,18 +73,111 @@ The geeky details of what you get:
   details='Policy:Calico,IPAM:Azure,CNI:Azure,Overlay:No,Routing:VPC Native,Datastore:Kubernetes'
 />
 
-Alternatively you can run in Azure CNI overlay mode.
+1. To create a cluster with Azure-managed Calico for policy, run the following command:
+  ```bash
```

Comment:
If you add a space before each line of the code block, it will show up in line with the step. You can see this aligned correctly in the next procedure.